pkgs/applications/misc/free42/default.nix

```diff
@@ -49,6 +61,7 @@ stdenv.mkDerivation rec {
     install -m644 gtk/README $out/share/doc/${pname}/README-GTK
     install -m644 README $out/share/doc/${pname}/README
 
+    install -m644 $desktopItem/share/applications/com.thomasokken.free42.desktop $out/share/applications
```

Comment:
Please use copyDesktopItems.

Reply:
Ah I didn't know there was a hook for that now. It should be fixed now.